README.md

```diff
@@ -419,11 +419,12 @@ To use subscription you can use either [subscriptions-transport-ws](https://gith
 
 `useSubscription(operation, callback)`
 
-- `operation`: Object - The GraphQL operation has the following properties:
+- `options`: Object - The GraphQL operation has the following properties:
```

Comment:
why did you rename this?

Reply:
I renamed because it didn't feel accurate calling it `operation` as it represents more than just the gql operation now.  It was a bit hasty though.  I'll rename it back to `operation`.

Reply:
options is also fine, but let's keep it consistent!

Reply:
Noted.

Reply:
I reverted to `operation` since `options` is being used in other contexts and changing would require more adjustments.